### PR TITLE
Technical advisory 69874 (urgent) (wait on timing for merge)

### PR DESCRIPTION
**Pull request by shermanCRL**
Relating to version upgrades

### CODE REVIEW

**Review by nickelnickel — APPROVED** [review]
LGTM

**Comment by shermanCRL** [comment]
Please don’t merge this PR until my (or someone else’s) go-ahead, needs to happen after the release has been removed.

File: releases/v21.1.8.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ summary: Additions and changes in CockroachDB version v21.1.8 since version v21.
 
 ## August 30, 2021
 
+{{site.data.alerts.callout_danger}}
+This patch release has been withdrawn due to [this technical advisory](../advisories/a69874.html).
+{{site.data.alerts.end}}
+
+
```
Comment (jseldess) [inline releases/v21.1.8.md:13]: Should we comment out everything below this warning? @ianjevans, thoughts?

Comment (shermanCRL) [reply releases/v21.1.8.md:13]: My two cents would be to keep it -- v21.1.8 is already out in the wild, so users should have access to have release notes.

Comment (jseldess) [reply releases/v21.1.8.md:13]: Ok, that's fine.

**Review by lnhsingh — COMMENTED** [review]
Extremely small nit, but otherwise LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/11232#-:-Mj15UgmCUTDttP2GA0n:b-px6jmh)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @celiala, @dt, and @shermanCRL)

___
*[advisories/a69874.md, line 15 at r2](https://reviewable.io/reviews/cockroachdb/docs/11232#-Mj15Nkh3jMNWyGzmHZQ:-Mj15Nkh3jMNWyGzmHZR:b-sq8b87) ([raw file](https://github.com/cockroachdb/docs/blob/9b6eeb64f532914c090ce0e20726bcff59514d76/advisories/a69874.md#L15)):*
> ```Markdown
> ## Statement
> 
> Patch releases are expected to maintain upgrade/downgrade/mixed-version compatibility, so a user can upgrade from 21.1.3 to 21.1.7 and back down to 21.1.5 at will.
> ```

nit: add `v` in front of version numbers 21.1.3 / 21.1.7 / 21.1.5

**Pull request merged**